lib/mainloop-control.c

```diff
@@ -69,20 +69,21 @@ control_connection_message_log(ControlConnection *cc, GString *command, gpointer
     g_string_assign(result, "FAIL Invalid arguments received");
 exit:
   g_strfreev(cmds);
-  return result;
+  return control_connection_send_reply(cc, result);
```

Comment:
We should not return from a void function (this is just a typo). Please remove the return keyword.

Reply:
I wanted to add `-Wreturn-type` as an extra warning flag, but it does not detect such typos without setting -Wpedantic :disappointed: .

Reply:
I've removed the extra return and force-pushed the branch, which should resolve this.

Reply:
Thanks.